src/components/layout/ShortHero.jsx

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
These components should not be in layouts folder. you can create your own folder and put your components

Reply:
I've made an about_page_component folder in the component folder to put my components